moveit_core/collision_detection/include/moveit/collision_detection/collision_detector_allocator.h

```diff
@@ -72,10 +72,7 @@ template <class CollisionEnvType, class CollisionDetectorAllocatorType>
 class CollisionDetectorAllocatorTemplate : public CollisionDetectorAllocator
 {
 public:
-  const std::string& getName() const override
-  {
-    return CollisionDetectorAllocatorType::NAME;
-  }
+  virtual const std::string& getName() const = 0;
```

Comment:
`getName()` is declared as abstract in the base class `CollisionDetectorAllocator` already. Why repeating this here?

Reply:
This is my fault.  I put this here to make it clear that this is also now a abstract class.  It would make no difference to delete it.

Reply:
We are happy either way; please let us know what your preference is.